### DISCUSSION TITLE
Exercism vs FreeCodeCamp

### DISCUSSION THREAD

**Discussion by kassyoveson** [USU-ITLS-x265-23-summer/class-repo#57] [Week 2 - Discussions]
Exorcism: This coding program reminds me of drinking from a fire house. There were a lot of elements presented at one time and you had to figure out what code to use based on the instructions. It would check the entire document every time to see if there were any coding errors, which can be helpful.  When there was an error, It didn't tell me why I got it wrong, it just put a ^ under where the mistake was. The program was very confusing and frustrating, so I YouTube a lot of it, so I don't know if I really learned much about coding.

FreeCodeCamp: This coding program was more user friendly. It walked you through step by step and gave you hints if you didn't pass the coding review. Sometimes it was confusing because it would give you 3-4 elements to change in a single step. I feel like I learned more coding through this program than exorcism. I would change it to have more examples of what the code should look like. There were times where I had to look to a third-party resource to figure out the order it wanted me to put it in.

**Comment by CJ3315** [comment]
Hello!
Yeah, I totally agree with the user-friendly aspect of both the camp and the exercises. I didn't really know what I was doing and there wasn't very much help when it came to what errors I was committing and such things. I was able to find a lot of helpful youtube tutorials which made things a bit clearer! Here is one that I spent a lot of time in: https://www.youtube.com/watch?v=qz0aGYrrlhU
